### PR TITLE
Feat: added badge and button Component

### PR DESCRIPTION
**Pull request by Nishant-96**
Feat: added Button and Badge. 

badge 2
Badge 10
button 1
button2
button3
button 4

### CODE REVIEW

File: Components/Badge/Badge.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+.icon-btn{
+    position: relative;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    width:50px;
+    height:50px;
```
Comment (znjs) [inline Components/Badge/Badge.css:8]: if possible give the sizing in relative units for responsivness

File: Components/Badge/Badge.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+.badge-av{
+    position: relative;
+    width: fit-content;
+
+}
+
+.profile-av{
+    width: 100px;
+    height: 100px;
```
Comment (znjs) [inline Components/Badge/Badge.css:42]: use relative units for responsiveness

File: Components/Badge/Badge.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    border-radius: 50%;
+    border:solid 0.2px;
+
+}
+
+.profile-av-pos{
+    top: -5px;
+    right: -5px;
+    width: 30px;
+    height:30px;
```
Comment (znjs) [inline Components/Badge/Badge.css:53]: if possible use relative units for responsiveness

**Review by znjs — APPROVED** [review]

**Review by JowelTisso — APPROVED** [review]

**Review by AnjaliDeshwani — APPROVED** [review]
Seems good. And agree with Jaya to use relative units.

**Review by anjali1102 — APPROVED** [review]
perfect

**Comment by Nishant-96** [comment]
Thank you for reviewing everyone. I made the changes as suggested.

**Pull request merged**